src/js/services/user-service.js

```diff
@@ -57,11 +57,9 @@ export function trackEvent(eventName, userProps) {
     const user = store.getState().user;
     return core().appUsers.trackEvent(user._id, eventName, userProps).then((response) => {
         if (response.conversationUpdated) {
-            return getConversation()
-                .then(connectFaye)
-                .then(() => {
-                    return response;
-                });
+            return handleConversationUpdated().then(() => {
```

Comment:
But the `GET /conversation` call can be avoided if faye is connected, because the message will come through websockets anyways